containersQa/testlib.bash

```diff
@@ -861,6 +861,11 @@ function checkOtherUserWorks() {
   test -z "$OUTPUT"
 }
 
+# This test is added to check that tar is installed in all Openjdk images.
+#  https://issues.redhat.com/browse/OPENJDK-2588
+function testTarIsInstalled() {
+  runOnBaseDir  tar --version
```

Comment:
in very extreme cases  (eg messed up security release of tar), the --version may not be enough , and `rpm -q tar` . But thats just an idea forfuture far far away. ty!